catalog.html

```diff
@@ -0,0 +1,322 @@
+            </a>
+            <a href="#" class="housing">Гостиница</a>
+            <a href="#" class="control">от 2000 ₽</a>
+
+            <form action="https://echo.htmlacademy.ru/" method="post"></form>
+            <button type="button" formtarget="housing-cards">Подробнее</button>
+            <button class="checkbox" type="button" formtarget="housing-list">В избранное</button>
+            <!---->
+            <form action="https://echo.htmlacademy.ru/" method="post"></form>
+            <label for="C5" class="label_rating"></label>
+            <input name="rating" id="C5" type="checkbox" checked>
+            <label for="C4" class="label_rating"></label>
+            <input name="rating" id="C4" type="checkbox" checked>
+            <label for="C3" class="label_rating"></label>
+            <input name="rating" id="C3" type="checkbox">
+            <label for="C2" class="label_rating"></label>
+            <input name="rating" id="C2" type="checkbox">
+            <label for="C1" class="label_rating"></label>
+            <input name="rating" id="C1" type="checkbox">
+            <button class="button" type="submit">Рейтинг<output name="sum"> 8,9 </output></button>
+
+          </li>
+
+          <li class="housing-cards">
+            <img src="" alt="Дорога-серпантин.">
+            <a class="housing-cards-link" href="housing.html">
+              <h3 class="housing-cards-title">GreenTree Inn</h3>
+            </a>
+            <a href="#" class="housing">Гостиница</a>
+            <a href="#" class="control">от 3000 ₽</a>
+
+            <form action="https://echo.htmlacademy.ru/" method="post"></form>
+            <button type="button" formtarget="housing-cards">Подробнее</button>
+            <button class="checkbox" type="button" formtarget="housing-list">В избранное</button>
+             <!---->
+            <form action="https://echo.htmlacademy.ru/" method="post"></form>
+            <label for="D5" class="label_rating"></label>
+            <input name="rating" id="D5" type="checkbox" checked>
+            <label for="D4" class="label_rating"></label>
+            <input name="rating" id="D4" type="checkbox" checked>
+            <label for="D3" class="label_rating"></label>
+            <input name="rating" id="D3" type="checkbox">
+            <label for="D2" class="label_rating"></label>
+            <input name="rating" id="D2" type="checkbox">
+            <label for="D1" class="label_rating"></label>
+            <input name="rating" id="D1" type="checkbox">
+            <button class="button" type="submit">Рейтинг<output name="sum"> 8,9 </output></button>
+          </li>
+        </ul>
+      </div>
+      <div class="addition">
+        <form action="https://echo.htmlacademy.ru/" method="post"></form>
+        <button class="pagination-show-more" type="button">Загрузить ещё</button>
+        <ul class="pagination">
+          <li class="pagination-item">
+            <a class="pagination-link" href="index.html">1</a>
```

Comment:
```suggestion
            <a class="pagination-link" href="">1</a>
```

нет, это не страницы сайта, это страницы для списка найденных гостиниц, как внизу поиска
![Снимок экрана 2022-02-03 в 10 58 19](https://user-images.githubusercontent.com/6417966/152303273-61964bfb-f59b-4c5c-ab5a-b364a7072fbf.jpg)